### DISCUSSION TITLE
Markdown images problem

### DISCUSSION THREAD

**Discussion by ejgutierrez74** [community/community#22602] [New to GitHub]
<p>I have a problem referencing a image in my wiki using mardown…<a href="https://github.com/ejgutierrez74/MyBilliards/wiki" rel="nofollow noopener">https://github.com/ejgutierrez74/MyBilliards/wiki</a></p>
<p>Here is my repo: <a href="https://github.com/ejgutierrez74/MyBilliards" rel="nofollow noopener">https://github.com/ejgutierrez74/MyBilliards</a></p>
<pre><code>Welcome to the MyBilliards wiki!

La meva primera imatge

Es veu:
![Billiards](https://github.com/ejgutierrez74/MyBilliards/blob/master/American(1024x585).jpg)

![Billiards JPG](../master/American(1024x585).jpg)
</code></pre>
<p>So i have an image in my root folder, if i get the url link in the first call, it works… But id prefer to work with local paths</p>
<p>like in the second call… Second call dont work and i have tried lots ( none of them works)…</p>
<p><img alt="Billiards JPG"></p>
<p><img src="/American(1024x585).jpg" alt="Billiards JPG"></p>
<p><img src="/blob/master/American(1024x585).jpg" alt="Billiards JPG"></p>
<p>I also would like to put all the images in a folder called documentImages and get all the images from it, just to mainteance and have some order.</p>
<p>Thanks</p>

**Comment by mpboom** [comment, marked as answer]
<p>I think</p>
<pre><code>[USERNAME]/[REPO]//blob/master/American(1024x585).jpg
</code></pre>
<p>will work.</p>

**Comment by ejgutierrez74** [comment]
<p>It worked</p>
<p>But its complicated as you can wrote a markdown document in local using ( image.jpg, local urls) and when you work/upload the documents to the wiki you have to update all the links using:</p>
<pre><code>[USERNAME]/[REPO]//blob/master/American(1024x585).jpg
</code></pre>
<p>If its not a bug, could be considered as a feature request, to use the same url in local and remote repos</p>
<p>Instead of:</p>
<pre><code>[USERNAME]/[REPO]//blob/master/American(1024x585).jpg
</code></pre>
<p> ( username, repo and blob, master can be added by default in images urls)</p>
<p>use:</p>
<pre><code>American(1024x585).jpg
</code></pre>

**Comment by AndreaGriffiths11** [comment]
<p>Hi Ejgutierrez74,</p>
<p>Thanks for this feedback! We’re always working to improve GitHub and the GitHub Community Forum, and we consider every suggestion we receive. I’ve logged your feature request in our internal feature request list.</p>
<p>Though I can’t guarantee anything or share a timeline for this, I can tell you that it’s been shared with the appropriate teams for consideration.</p>
<p>Please let me know if you have any other questions.</p>
<p>Cheers!</p>
<p>Andrea</p>

**Comment by mpboom** [comment]
<p>Could you try</p>
<pre><code>../blob/master/American(1024x585).jpg
</code></pre>
<p>? It should work as well an doesn’t require a username/repo in the URL.</p>

**Comment by ejgutierrez74** [comment]
<p>This one doesnt work for me, but still think that to be in the same line and avoid some extra-work, the natural way is writing the same code locally than the wiki…</p>
<p>So ( images/American.jpg) should work and Github add USERNAME, REPO, blob, master, whatever is needed… ( think of an assignment with 100 images or more…) It seems not very difficult to implement this. Just a thought and a request.</p>
<p>Thank for your work and help</p>

**Comment by ejgutierrez74** [comment]
<p>Thanks, keep me updated</p>
